node/config/def.go

```diff
@@ -66,6 +66,20 @@ type DealmakingConfig struct {
 
 	Filter          string
 	RetrievalFilter string
+
+	// If UseDynamicRetrievalPricing is set to false(default), we will use the DefaultRetrievalPricingParams
+	// for pricing retrieval deals, otherwise we will use the DynamicRetrievalPricingFunc configured here.
+	UseDynamicRetrievalPricing  bool
+	DynamicRetrievalPricingFunc string
+
+	DefaultRetrievalPricingParams DefaultRetrievalPricingParams
+}
+
+type DefaultRetrievalPricingParams struct {
+	PricePerByte            types.FIL
+	UnsealPrice             types.FIL
+	PaymentInterval         uint64
+	PaymentIntervalIncrease uint64
```

Comment:
Let's keep the set retrieval ask command and remove this from configuration. We can't break that interface. 

That allows for setting a "fixed" ask for when:

1. clients are querying the retrieval ask (get retrieval ask)
2. a dynamic pricing function (if enabled) refuses to calculate a dynamic price, or it errors

And miners will want to adjust the fixed price at runtime, so having these in configuration removes flexibility.